### PR TITLE
Add toastAndroid module bindings from Rebolt.

### PR DESCRIPTION
**Pull request by Trancever**
Add toastAndroid module bindings from Rebolt.

### CODE REVIEW

File: src/toastAndroid.rei [hunk 1]
```diff
@@ -0,0 +1,17 @@
+type duration =
+  | BOTTOM
+  | CENTER
```
Comment (wokalski) [inline src/toastAndroid.rei]: is duration `CENTER` really a thing :D ?

File: src/toastAndroid.re [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+[@bs.module "react-native"] [@bs.scope "ToastAndroid"]
+external show : (string, duration) => unit = "";
+
+[@bs.module "react-native"] [@bs.scope "ToastAndroid"]
+external showWithGravity : (string, duration, gravity) => unit = "";
+
+[@bs.module "react-native"] [@bs.scope "ToastAndroid"]
+external showWithGravityAndOffset :
+  (string, duration, gravity, int, int) => unit =
```
Comment (wokalski) [inline src/toastAndroid.re]: do we really pass the variants as is to JS?

File: src/toastAndroid.rei (hunk 1 above)
Comment (Trancever) [reply src/toastAndroid.rei]: Fixed :D

File: src/toastAndroid.re (hunk 2 above)
Comment (Trancever) [reply src/toastAndroid.re]: Yeah, bindings were totally broken, fixed them already so you can check once again :)

**Comment by wokalski** [comment]
It looks good to me although I believe we should use labels for the x/y offsets.

**Comment by grabbou** [comment]
I have updated the interface. Unfortunately, due to really bad API from the React Native side (multiple `show` methods with no clear default parameter), I was unable to provide a single, unified, entry-point with optional labels.

Decided to follow the Animated API where the first argument is provided and rest that follows are labels. I think it matches the overall API pretty nice.

**Comment by grabbou** [comment]
We could provide a unified `show` method but in case the user provides offset and no gravity, there's no native React Native method to call. In that case, we would have to assume the default value for "gravity" which is unclear and not described in React Native docs. Question is, whether we should make such default for better API sake?

Essentially, it would be a single switch/case statement checking on the arguments provided and calling the appropriate native method.

**Comment by grabbou** [comment]
Here's the method with alternative API:
```reason
let betterShow = (message, ~duration, ~gravity=?, ~offset=?, ()) =>
	switch ((gravity, offset)) {
	| (None, None) => _show(message, getDuration(duration))
	| (Some(gravity), None) => _showWithGravity(message, getDuration(duration), getGravity(gravity))
	| (Some(gravity), Some((x, y))) => _showWithGravityAndOffset(message, getDuration(duration), getGravity(gravity), x, y)
	| (None, Some((x, y))) => _showWithGravityAndOffset(message, getDuration(duration), getGravity(CENTER), x, y)
	}
```
Note that the last case assumes "`CENTER`" is the default when offset is set. We don't have `showWithOffset` method on the native side and so, we need to put a default.

**Pull request merged**